packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,439 @@
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="bottom" shouldFlip={false}>
+  <ActionButton>
+      align=end direction=bottom
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="top" shouldFlip={false}>
+  <ActionButton>
+      align=end direction=top
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Autofocus
+
+Applying `autoFocus` to the Menu of the MenuTrigger sets focus within the Menu upon opening.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      autofocus
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    selectedKeys={['Kangaroo']}
+    autoFocus>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger>
+  <ActionButton>
+      autofocus=false
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    selectedKeys={['Kangaroo']}
+    autoFocus={false}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger>
+  <ActionButton>
+      autofocus=first
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus="first">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger>
+  <ActionButton>
+      autofocus=last
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus="last">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Closes Menu onSelect
+
+```tsx example
+<MenuTrigger closeOnSelect>
+  <ActionButton>
+      closeOnSelect=true
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      closeOnSelect=false
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Disabled Trigger
+
+```tsx example
+<MenuTrigger>
+  <ActionButton isDisabled>
+      Disabled Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Disabled Menu Items
```

Comment:
this is a good one, i like this one because it's explaining menu items in the menu documentation
maybe we should just not mention the disabled actionbutton and leave it to the button documentation